pandas/tests/extension/base/methods.py

```diff
@@ -103,3 +103,36 @@ def test_factorize_equivalence(self, data_for_grouping, na_sentinel):
 
         tm.assert_numpy_array_equal(l1, l2)
         self.assert_extension_array_equal(u1, u2)
+
+    def test_combine_le(self, data_repeated):
```

Comment:
can you give a 1-liner explaining what this is testing. the name of the test is uninformative.

Reply:
done